src/config/konnectors.json

```diff
@@ -1561,6 +1561,32 @@
       "bankId": "119"
     }
   },
+  {
+    "slug": "impots",
+    "name": "Impots.gouv",
+    "editor": "Cozy Cloud",
+    "vendorLink": "https://cfspart.impots.gouv.fr/",
+    "categories": ["public_service"],
+    "dataType": [
+      "public_service"
+    ],
+    "fields": {
+      "identifier": {
+        "type": "text"
+      },
+      "password": {
+        "type": "password"
+      },
+      "advancedFields": {
+        "folderPath": {
+          "advanced": true,
+          "isRequired": false
+        }
+      }
+    },
+    "testing": "true",
```

Comment:
I think adding `"frequency": "daily"`, was a good idea to allow the test of the trigger for the next day. We can remote it when we also remove `testing: true` later.
`

Reply:
Done.